contracts/src/bridge/IBridge.sol

```diff
@@ -2,19 +2,37 @@
 
 pragma solidity >=0.7.0 <0.9.0;
 
+// The ERC20 token bridge interface.
+// Calling functions on it will result in assets being bridged over to the other layer automatically.
 interface IBridge {
-    struct TransferMessage {
-        address asset;
-        uint256 amount;
-        address target;
-    }
-
     enum Topics {
         TRANSFER,
         MANAGEMENT
     }
 
-    function sendNative(address target) external payable;
-    function sendAssets(address asset, uint256 amount, address receiver) external;
-    function receiveAssets(address asset, uint256 amount, address receiver) external;
-}
\ No newline at end of file
+    // Sends the native currency to the other layer. On Layer 1 the native currency is ETH, while on Layer 2 it is OBX.
+    // When it arrives on the other side it will be wrapped as a token.
+    // receiver - the L2 address that will receive the assets on the other network.
+    function sendNative(address receiver) external payable;
+
+    // Sends ERC20 assets over to the other network. The user must grant allowance to the bridge
+    // bofore calling this function for more or equal to the amount being bridged over. 
+    // This can be done using IERC20(asset).increaseAllowance(bridge, amount); 
```

Comment:
`increaseAllowance` is not ERC20 compliant. I think it's an openzepellin added method.

Reply:
Nice catch